core/tests/webdriverio_utils/ExplorationEditorPage.js

```diff
@@ -408,8 +408,10 @@ var ExplorationEditorPage = function() {
   };
 
   this.waitForPreviewTabToLoad = async function() {
-    await waitFor.visibilityOf(
-      previewTabLearnerViewCard, 'Preview Tab learner card is not visible');
+    // We need to use browser.pause in order to wait for preview tab
```

Comment:
`waitFor.pageToFullyLoad` only works when we navigate to preview tab for the first time, on the next time we need to explicitly wait using browser.pause because even adding `visibilityOf` is not working and causing this flake https://github.com/oppia/oppia/actions/runs/3312048155/jobs/5468246555

The screenshot shows that the click happens on the main tab icon but the navigation does not happens because preview tab takes some time to load.
![Full-exploration-editor-should-delete-interactions-cleanly](https://user-images.githubusercontent.com/86483059/198314456-7a35f2f1-7505-45cc-b921-432da11c38fe.png)

I was even able to reproduce this on server, the first time we are getting a loading screen but second time no loading screen but click does not work if we click on main tab real quick (webdriverio is too fast)

https://user-images.githubusercontent.com/86483059/198315033-ec89377b-2058-4c0b-85f7-4df5badee081.mp4

Reply:
Is it a frontend bug that when switching to the main tab too quickly, the page fails to finish loading?

Reply:
Yes, because switching to all other tab works well though user is not supposed to click this much fast so can't say we need to fix this.